### ISSUE TITLE
Errors in R chapter

### ISSUE THREAD

**Issue by jtr13** [jtr13/cc21#81]
I see all the code chunks were set to `eval = FALSE` in this chapter: https://jtr13.github.io/cc21/common-errors-in-r.html. Not a top priority but it would be better if we could get it to work without that. The chunk option `error = TRUE` should allow the document to render with the errors showing, which is the intention here since the chapter is about errors. I'm not sure why that's not working since they have it set globally in this chunk:

    ```{r, include=FALSE}
    knitr::opts_chunk$set(echo = TRUE, error = TRUE)
    ```

In short, at some point let's look into this.

**Comment by kennylee15** [comment]
Last night, I noticed the intentional mistakes in those codes were causing build failures. I understood the intention of the document.

Adding `eval=FALSE` was my solution to avoid the build failure. Let me try again sometime today to address this problem.

**Comment by jtr13** [comment]
It's strange that this is happening since it works locally with `error=TRUE` ???

**Comment by kennylee15** [comment]
I noticed that certain errors cannot be shown as output without halting R for some reason even though `error = TRUE` is included for all chunks and they run as they should locally with RStudio. 

These errors include missing parentheses, forgotten operators, and `condition has length > 1 ` errors. 

For these chunks, `eval = FALSE ` was added. I manually added in the error message for these chunks.

**Comment by kennylee15** [comment]
https://jtr13.github.io/cc21/common-errors-in-r.html

It seems good to go now!
